app/models/ability.rb

```diff
@@ -286,7 +289,14 @@ def orders_process_checklists_abilities
 
   def organisations_abilities
     if can_check_organisations? || @api_user
-      can [:index, :search, :show, :orders], Organisation
+      can %i[index search show orders], Organisation
+    end
+    can %i[create update], Organisation if can_manage_organisation? || @api_user
+  end
+
+  def organisation_type_abilities
+    if can_access_organisation_types? || @api_user
+      can [:index], OrganisationType
     end
   end
```

Comment:
Can't we make this public ? If I'm not mistaken they already are visible in the repo's yaml files anyways